README.md

```diff
@@ -26,69 +26,70 @@ Install the Docker generator:
 npm install -g generator-docker
 ```
 
-Run the generator in the same folder that your project is in:
+## Getting Started
+Once yo docker and VS Code are installed, here's the few steps to debug your first app
+### Node.js
+- From the root of your project, where your server.js file existts, open a terminal window
```

Comment:
exists: fixed
The scaffolding for the ENTRYPOINT uses server.js. 
`ENTRYPOINT ["node", "server.js"]`
Changed text to powershell/bash
removed $'s